src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -880,9 +880,15 @@ private void UpdateTopResult(SearchMemberGroup memberGroup)
             }
 
             var topMemberGroup = new SearchMemberGroup(memberGroup.FullyQualifiedName);
-            topMemberGroup.AddMember(memberGroup.Members.First());
 
+            // Clone top node.
+            var topNode = new NodeSearchElementViewModel(memberGroup.Members.FirstOrDefault());
```

Comment:
We need to have clone here, because of this situation: 
![image](https://cloud.githubusercontent.com/assets/8158404/9114333/f3fab98c-3c61-11e5-9477-74effced7414.png)

Since top result was one of search members. It used to updated with search member. I.e. is search member is selected, top result selected as well.